src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -75,14 +75,18 @@ void InitCapabilities() {
 
   tts_enum_capabilities.insert(std::make_pair(
       std::string("TEXT"), hmi_apis::Common_SpeechCapabilities::SC_TEXT));
-  tts_enum_capabilities.insert(std::make_pair(
-      std::string("SAPI_PHONEMES"), hmi_apis::Common_SpeechCapabilities::SAPI_PHONEMES));
-  tts_enum_capabilities.insert(std::make_pair(
-      std::string("LHPLUS_PHONEMES"), hmi_apis::Common_SpeechCapabilities::LHPLUS_PHONEMES));
-  tts_enum_capabilities.insert(std::make_pair(
-      std::string("SAPI_PHONEMES"), hmi_apis::Common_SpeechCapabilities::SAPI_PHONEMES));
-  tts_enum_capabilities.insert(std::make_pair(
-      std::string("PRE_RECORDED"), hmi_apis::Common_SpeechCapabilities::PRE_RECORDED));
+  tts_enum_capabilities.insert(
+      std::make_pair(std::string("SAPI_PHONEMES"),
+                     hmi_apis::Common_SpeechCapabilities::SAPI_PHONEMES));
+  tts_enum_capabilities.insert(
+      std::make_pair(std::string("LHPLUS_PHONEMES"),
+                     hmi_apis::Common_SpeechCapabilities::LHPLUS_PHONEMES));
+  tts_enum_capabilities.insert(
+      std::make_pair(std::string("SAPI_PHONEMES"),
+                     hmi_apis::Common_SpeechCapabilities::SAPI_PHONEMES));
```

Comment:
Not in the scope of this PR, but there is a duplicate of the `SAPI_PHONEMES` value that is added to this `tts_enum_capabilities` list. Shouldn't cause any issues, just a cosmetic issue overall.